### PR TITLE
[13.0][MIG] sale_financial_risk

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

- [x] Depends on https://github.com/OCA/credit-control/pull/67.

### CODE REVIEW

File: sale_financial_risk/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    risk_sale_order = fields.Monetary(
+        compute="_compute_risk_sale_order",
+        string="Total Sales Orders Not Invoiced",
+        help="Total not invoiced of sales orders in Sale Order state",
+    )
+
+    def _get_risk_sale_order_domain(self):
+        # When p is NewId object instance bool(p.id) is False
+        commercial_partners = self.filtered(
+            lambda p: (p.customer and p.id and p == p.commercial_partner_id)
```
Comment (carlosdauden) [inline sale_financial_risk/models/res_partner.py]:
```suggestion
            lambda p: (p.customer_rank and p.id and p == p.commercial_partner_id)
```

File: sale_financial_risk/tests/test_partner_sale_risk.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+from odoo.tests.common import SavepointCase
+
+
+class TestPartnerSaleRisk(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestPartnerSaleRisk, cls).setUpClass()
+        cls.env.user.groups_id |= cls.env.ref("sales_team.group_sale_manager")
+        cls.partner = cls.env["res.partner"].create(
+            {"name": "Partner test", "customer": True}
```
Comment (carlosdauden) [inline sale_financial_risk/tests/test_partner_sale_risk.py]:
```suggestion
            {"name": "Partner test", "customer_rank": 1}
```

**Review by carlosdauden — CHANGES_REQUESTED** [review]

**Comment by carlosdauden** [comment]
Can you rebase? 
https://github.com/OCA/credit-control/pull/67 is merged.

**Comment by MiquelRForgeFlow** [comment]
I already did. If not, travis wouldn't be green. :)

**Comment by carlosdauden** [comment]
Ups, sorry.
I have this tab open and it seems that my browser had not refreshed well because I was still seeing it red.

**Review by carlosdauden — APPROVED** [review]

**Comment by carlosdauden** [comment]
In fact I had been surprised that the check of the first comment did not appear marked...

**Comment by pedrobaeza** [comment]
Should we use then this one instead of respecting @sergio-teruel's first PR?

**Comment by sergio-teruel** [comment]
For me Ok... no problem...,

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**